playbooks/streisand-client.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I had intentions of changing the playbook/role name but wasn't able to think of anything great. I agree that it's likely to cause confusion. I'd like to get "test" in there somehow. I don't have time to tackle these revisions for another day or so, I'll strew on this and try to think of something.